caluma/caluma_core/tests/test_health_checks.py

```diff
@@ -0,0 +1,542 @@
+    This affects the watchman check for databases and the db models
+    check and prevents access to the database.
+    """
+
+    # use database fixture if test in working state
+    if working_db:
+        request.getfixturevalue("db")
+        # manually migrate db to ensure the database is in the
+        # newest state when running migration tests database
+        # might be left in an inconsistent state
+        management.call_command("migrate")
+
+    # get /healthz/ response and compare to previous snapshot
+    response = client.get("/graphql/healthz")
+    snapshot.assert_match(response.json())
+
+    # assert database check response
+    if working_db:
+        assert response.json()["databases"] == [{"default": {"ok": True}}]
+        assert response.json()["db models"] == [{"default": {"ok": True}}]
+        assert response.json()["db migrations"] == [{"default": {"ok": True}}]
+        assert response.status_code == 200
+    else:
+        assert response.json()["databases"] == [{"default": {"ok": False}}]
+        assert response.json()["db models"] == [{"default": {"ok": False}}]
+        assert response.json()["db migrations"] == [{"default": {"ok": False}}]
+        assert response.status_code == watchman_settings.WATCHMAN_ERROR_CODE
+
+
+def switch_user(user, pwd):
+    """
+    Switch database user to either 'tester' or 'caluma' and
+    re-connect to database
+    """
+
+    # check if valid user
+    users = {"tester", "caluma"}
+    if not user in users:  # pragma: no cover
+        raise Exception("Could not switch user: Invalid user name " + user)
+
+    # connect new user to database
+    connections["default"].close()
+
+    settings.DATABASES["default"]["USER"] = user
+    settings.DATABASES["default"]["PASSWORD"] = pwd
+
+    connections["default"].connect()
+
+
+def switch_user_tester():
+    """
+    Switch database user to 'tester' with password 'test'
+    """
+
+    switch_user("tester", "test")
+
+
+def switch_user_caluma():
+    """
+    Switch database user to 'tester' with password 'test'
+    """
+
+    switch_user("caluma", "caluma")
+
+
+def create_test_user():
+    """
+    Create new user 'tester' with privileges necessary for health checks
+    """
+
+    # get db connection
+    cursor = connections["default"].cursor()
+
+    # create user and grant privileges
+    cursor.execute("CREATE USER tester WITH PASSWORD 'test';")
+    cursor.execute("GRANT ALL ON caluma_logging_accesslog TO tester;")
+    cursor.execute(
+        "GRANT USAGE, SELECT ON SEQUENCE caluma_logging_accesslog_id_seq TO tester;"
+    )
+    cursor.execute("GRANT ALL ON django_migrations TO tester;")
+
+
+def remove_test_user():
+    """
+    Remove user 'tester' and revoke privileges for health checks
+    """
+
+    # get db connection
+    cursor = connections["default"].cursor()
+
+    # revoke privileges and remove user
+    cursor.execute("REVOKE ALL ON caluma_logging_accesslog FROM tester;")
```

Comment:
No need to `REVOKE` - this happens implicitly when dropping the user

Reply:
When the `REVOKE` statements are removed the tests trigger a `role "tester" cannot be dropped because some objects depend on it` error with reference to the privileges that weren't revoked explicitly. Is there a command that handles the dropping of those privileges automatically?